pybaseball/fielding_leaders.py

```diff
@@ -0,0 +1,4 @@
+from .datasources.fangraphs import fg_fielding_data
+
+# This is just a pass through for the new, more configurable function
+fielding_stats = fg_fielding_data
```

Comment:
Do we need a file for this?

Reply:
Need? No. I was just trying to be consistent with the other two (batting_stats, pitching_stats).